CHANGELOG.md

```diff
@@ -26,3 +26,8 @@ New modules
 
 Additions to existing modules
 -----------------------------
+
+* New lemma in `Data.Vec.Properties`:
+  ```agda
+  map-concat : map f (concat xss) ≡ concat (map (map f) xss)
```

Comment:
aaaargh! This type has the opposite orientation to
```agda
Data.List.Properties.concat-map : ∀ {f : A → B} → concat ∘ map (map f) ≗ map f ∘ concat
```

Suggest reversing the orientation for `Vec`, for consistency's sake...

Reply:
Or else: a `breaking` refactoring of the `List` definition... :-(

Reply:
Aaargh... ignore! `map-concat` vs. `concat-map`... sigh :man_facepalming: